src/nameserver/name_server_impl.cc

```diff
@@ -5583,10 +5583,25 @@ int NameServerImpl::CreateChangeLeaderOPTask(std::shared_ptr<OPData> op_data) {
 
 void NameServerImpl::OnLocked() {
     PDLOG(INFO, "become the leader name server");
-    bool ok = Recover();
-    if (!ok) {
+    if (!Recover()) {
         PDLOG(WARNING, "recover failed");
     }
+    if (IsClusterMode() && (db_table_info_.empty() || db_table_info_[INTERNAL_DB].empty())) {
+        if (tablets_.size() < FLAGS_system_table_replica_num) {
+            LOG(FATAL) << "tablet num " << tablets_.size()
+                << " is less then system table replica num " << FLAGS_system_table_replica_num;
+            exit(1);
+        }
+        auto status = CreateDatabase(INTERNAL_DB);
+        if (!status.OK() && status.code != ::openmldb::base::ReturnCode::kDatabaseAlreadyExists) {
+            LOG(FATAL) << "create internal database failed";
+            exit(1);
+        }
+        if (FLAGS_system_table_replica_num > 0 && !CreateSystemTable("JOB_INFO", SystemTableType::kJobInfo).OK()) {
```

Comment:
Should we define the table name "JOB_INFO" here?  or "JOB_INFO" should be a constant?